cpp/include/raft/sparse/convert/csr.cuh

```diff
@@ -108,19 +108,21 @@ void adj_to_csr(raft::resources const& handle,
 /**
  * @brief Converts a bitmap matrix into unsorted CSR format matrix.
  *
- * @tparam       bitmap_t    Underlying type of the bitmap.
- * @tparam       index_t     Indexing type used.
- * @tparam       value_t     Data type of CSR
- * @tparam       nnz_t       Type of CSR
+ * @tparam       bitmap_t         Underlying type of the bitmap.
+ * @tparam       index_t          Indexing type used.
+ * @tparam       csr_matrix_t     Reference Type of CSR Matrix, raft::device_csr_matrix
  *
- * @param[in]    handle      RAFT handle
- * @param[in]    bitmap      input raft::bitmap_view
- * @param[out]   csr         output raft::device_csr_matrix_view
+ * @param[in]    handle           RAFT handle
+ * @param[in]    bitmap           input raft::bitmap_view
```

Comment:
In and out are already specified on the param so we shouldn't need to duplicate those in the description. Can you please expand the descriptions, though? Instead of just "handle" and "raft::bitmap_view", it would be helpful to explain their purpose. 

Also, please don't include "reference type" in the description. The caller doesn't have to care that it's a reference type. That's automatic when they pass in the argument.

Reply:
I think we should still be accepting [device_structure_owning_csr_matrix](https://github.com/rapidsai/raft/blob/3e67a6c3bc5c73cd4a77a71bb5c2d911f8a7bcd4/cpp/include/raft/core/device_csr_matrix.hpp#L74) here unless we intend to explicitly support both. If a user passes in a structure-preserving csr matrix, they will end up getting an error when you try to initialize the sparsity.

Reply:
Make sense.